remme/rpc_api/transaction.py

```diff
@@ -32,7 +32,7 @@
 
 __all__ = (
     'send_raw_transaction',
-    'send_tokens',
+    # 'send_tokens',
```

Comment:
Why?

Reply:
It just for test, I forgot about removing of it from base methods.